README.md

```diff
@@ -28,32 +31,80 @@ The decision to fork the original mender-mcu-client was made to:
 * **Provide Official Support**: Ensure that the project receives the necessary attention and
   resources from Northern.tech to meet the needs of the community and enterprise users.
 
+## Getting started
 
-## Get Started
-
-Since the project is under active development, we recommend watching the repository or checking back
-regularly for updates. Detailed documentation and usage instructions will be provided as the project
-progresses.
-
+For using `mender-mcu` for the first time, we strongly recommend you start with our [Zephyr reference application](https://github.com/mendersoftware/mender-mcu-integration).
 
-## Get started with Zephyr OS
-
-For using `mender-mcu` for the first time, we strongly recommend you start with our Zephyr reference
-application.
-
-It can be found in the repository
-[mender-mcu-integration](https://github.com/mendersoftware/mender-mcu-integration). The policy is
-that the reference application's `main` branch will follow the latest stable branch of `mender-mcu`
+The policy is that the reference application's `main` branch will follow the latest stable branch of `mender-mcu`
 (this repository).
 
-In the `mender-mcu-repository` you will find:
+In the `mender-mcu-integration` repository you will find:
 * A reference `west` workspace for Zephyr OS builds, including `CMake` and `KConfig` configurations.
 * Sample application for how to use `mender-mcu` APIs.
 * A list of boards that we have so far tested.
 
 We recommend starting there to understand how we have designed our solution and then coming back
 here for more in-depth information.
 
+### Compatibility
+| Zephyr OS version | Bootloader |
+|-------------------|------------|
+| v3.7.0            | MCUboot    |
+
+### Boards
+The reference board for `mender-mcu` is the [ESP32-S3-DevKitC](https://docs.espressif.com/projects/esp-dev-kits/en/latest/esp32s3/esp32-s3-devkitc-1/index.html#).
+
+To build the project, you need a board that supports MCUboot. The following link will filter
+the officially supported boards that also support MCUboot:
+
+* [Zephyr Project supported boards with MCU boot](https://docs.zephyrproject.org/latest/gsearch.html?q=MCUboot&check_keywords=yes&area=default#gsc.tab=0&gsc.q=MCUboot&gsc.ref=more%3Aboards&gsc.sort=)
+
+
+### Setting up a West project with `mender-mcu` as a module
+See the [Zephyr documentation](https://docs.zephyrproject.org/latest/develop/getting_started/index.html) for more information
+on getting started with Zephyr.
+
+In order to use `mender-mcu` as a [module](https://docs.zephyrproject.org/latest/develop/modules.html#modules-external-projects)
+in a [West project](https://docs.zephyrproject.org/latest/develop/west/basics.html#west-workspace), you will
+need to add `mender-mcu` to your project's west manifest:
+
+```yaml
+manifest:
+  remotes:
+  #...
+    - name: mender
+      url-base: https://github.com/mendersoftware
```

Comment:
This is not necessary if we use `url` in the module. I think it is cleaner to remove the `remotes`, as we expect only one module to be needed, but also specially here in the README for a shorter snippet.